.github/workflows/tests.yml

```diff
@@ -27,7 +34,16 @@ jobs:
         if: steps.npm-cache.outputs.cache-hit != 'true'
         run: npm ci
 
-      - name: Run tests
-        run: npx percy exec -- npm test
+      - name: Build assets
+        run: npm run build:assets
+
+      - name: Run test projects
```

Comment:
My only slight reservation about this is that it if we add new test projects we need to remember to come and update this.

But this might just be my own mental model that CI checks ~= npm test.

Reply:
From conversations on Slack I think the intent is to use the different jobs to parallelise the tests, which depends on this change, so happy to continue as is.

Reply:
Yeah I agree with you. I'll share a screenshot though

<img width="658" alt="Runners" src="https://user-images.githubusercontent.com/415517/190686393-4d4c3196-c896-4d24-9609-18b583b4280f.png">

Having separate (named) Jest projects means we can parallelise them 😮 

Our 2.5 minute test run goes down to about ~40 seconds total